### DISCUSSION TITLE
Please generate stable markup

### DISCUSSION THREAD

**Discussion by TWolversonReply** [Azure/logicapps#1127] [Feature Ideas/Requests]
The VSCode designer and the Portal designer rearrange workflow markup seemingly at random, causing horrendous source control churn for only minor functional changes, making it very difficult to code review. Typically whenever a developer on my team makes any change at all to a workflow using the designer, every property under every action changes places. I haven't spotted any pattern to this, but it's:
- any and all workflows, regardless of their contents i.e. it always happens
- doesn't seem to tie to any particular version of the vscode extension
- doesn't seem to tie to any particular user's setup i.e. it happens for everybody

**Comment by karl-lonnberg** [comment]
This issue has been around for years ( https://github.com/Azure/logicapps/issues/265 ) so let's fix it! :)

The same workflow created in VSCode, in the portal designer or if you download the logic app zip, you can get three different formats. So which formatting would you keep? How about sorting alphabetically and set 2-space-intend on save?

**Reply by TWolversonReply** [reply]
I don't think it matters in the slightest what convention is adopted as long as it's consistent everywhere.

**Comment by TWolversonReply** [comment]
Any update?
